clients/shared/table_config.go

```diff
@@ -46,7 +48,16 @@ func GetTableConfig(args GetTableCfgArgs) (*types.DwhTableConfig, error) {
 		return tableConfig, nil
 	}
 
-	rows, err := args.Dwh.Query(args.Query)
+	var rows *sql.Rows
+	var err error
+
+	// TODO: Get everyone to pass in args.
+	if len(args.Args) > 0 {
```

Comment:
I think these two branches may be equivalent.

Reply:
[[]string{}](https://go.dev/play/p/UFkPiQN6ONj)

Reply:
Great!